jenkins/catapult/runDockerTestsInnerLint.py

```diff
@@ -110,8 +112,8 @@ def main():
 	environment_manager.chdir(script_path)
 
 	linter_runner = LinterRunner(process_manager, args.out_dir, args.dry_run)
-	run_shell_linters(linter_runner, find_files_with_extension(environment_manager, '.sh'))
-	run_python_linters(linter_runner, find_files_with_extension(environment_manager, '.py'), str(script_path))
+	run_shell_linters(linter_runner, find_files_with_extension(environment_manager, source_dir, '.sh'))
+	run_python_linters(linter_runner, find_files_with_extension(environment_manager, source_dir, '.py'), str(script_path))
```

Comment:
these were originally scanning files in /scripts (catapult-src/jenkins/catapult) which is wrong directory and indeed has no sh files

Reply:
Should these linters run in the PR flow?